pkgs/development/python-modules/pysonos/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, xmltodict
+, requests
+, ifaddr
+}:
+
+buildPythonPackage rec {
+  pname = "pysonos";
+  version = "0.0.12";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1pfm23v2mj8l878ilpy9xb0jbkpncnwyf4432haraswvf1cg2bxh";
+  };
+
+  propagatedBuildInputs = [ xmltodict requests ifaddr ];
+
+  # Need to override pytest-conv to an old version
+  doCheck = false;
+
+  meta = {
+    homepage = https://github.com/amelchio/pysonos;
+    description = "A SoCo fork with fixes for Home Assistant.";
+    license = lib.licenses.mit;
+  };
```

Comment:
`maintainers` is missing

Reply:
You need to add yourself to `maintainers/maintainer-list.nix`.

Reply:
done